src/interface.jl

```diff
@@ -64,21 +66,23 @@ Returns a `DensityInterface`-compatible density that is defined by a given
 log-density function `log_f`:
 
 ```julia
```

Comment:
Could we use doctests?

Reply:
Hm, we'd need a concrete test case, then, I wanted to to avoid that here, to not limit the users idea of what `d` could be.

Reply:
Oh, wait - we can set up the test `d` invisibly - I'll do that!